db/seeds.rb

```diff
@@ -29,11 +31,19 @@ def generate_diets
 		end
 	end
 
+	def generate_cuisines
+		["American", "Italian", "Asian", "Mexican", "Southern & Soul Food", "French", "Southwestern", "Barbecue", "Indian", "Chinese", "Cajun & Creole", "English", "Mediterranean", "Greek", "Spanish", "German", "Thai", 
+				"Moroccan", "Irish", "Japanese", "Cuban", "Hawaiin", "Swedish", "Hungarian", "Portugese"].each do |cuisine|
```

Comment:
Line is too long. [111/80]<br>Align the elements of an array literal if they span more than one line.<br>Tab detected.